editor/src/components/filebrowser/fileitem.tsx

```diff
@@ -205,6 +205,20 @@ const isFile = (fileBrowserItem: FileBrowserItemInfo) => {
   return fileBrowserItem.type === 'file'
 }
 
+export const getGithubFileStatusColor = (type: GithubFileStatus): string => {
+  // NOTE: these are placeholder colors, we should update them once we finalize the design
```

Comment:
Don't worry about it for the sake of this PR since the design is still a WIP, but we have colour themes in [`theme.ts`](https://github.com/concrete-utopia/utopia/blob/master/editor/src/uuiui/styles/theme.ts), and a hook [`useColorTheme`](https://github.com/concrete-utopia/utopia/blob/master/editor/src/uuiui/styles/theme.ts#L314) to access the current theme in functional components.

Reply:
noice 👌